script.js

```diff
@@ -8,42 +8,33 @@ hambuger.addEventListener('click', () => {
   closeBtn.classList.toggle('active');
 });
 
-// validation form
+// Retrieve saved data from localStorage and fill in form fields
 const form = document.getElementById('form-input');
 const email = document.getElementById('email');
 const fullName = document.getElementById('name');
 const message = document.getElementById('comment');
 const errorMsg = document.getElementById('error');
 const regex = /^[a-z0-9._-]+@[a-z0-9.-]+\.[a-z]{2,4}$/g;
 
-// Check if there's a saved email in localStorage
-const savedEmail = localStorage.getItem('email');
-if (savedEmail) {
-  email.value = savedEmail;
-}
+// Helper function to save data in localStorage
 
-// Check if there's a saved full name in localStorage
-const savedName = localStorage.getItem('fullName');
-if (savedName) {
-  fullName.value = savedName;
+function saveData() {
+  const formData = {
+    fullname: fullName.value,
+    email: email.value,
+    message: message.value,
+  };
+  localStorage.setItem('formInputData', JSON.stringify(formData));
 }
 
-// Check if there's a saved first Message in localStorage
-const savedMsg = localStorage.getItem('message');
-if (savedMsg) {
-  message.value = savedMsg;
-}
+form.addEventListener('submit', saveData);
```

Comment:
Please kindly implement these requirements 💪

![image](https://github.com/aradradev/abdulsportfolio/assets/78251927/5f453b14-0fc3-4cf5-9db9-01d9622b097e)

### Hints 🤗
- The contact form information is persistent to page reloads.
- After filling the form, `and not submitting it`, the entered values should be saved in the browser.
- When visiting the page the form inputs should be contained, and be auto-populated with, ***the values entered in the previous visit to the page.***
- If at the previous visit to the page, in the same browser, no input was entered in the form then the form inputs should remain empty.